graph/src/components/networks/mod.rs

```diff
@@ -0,0 +1,76 @@
+use std::convert::TryFrom;
+use std::fmt;
+use std::sync::Arc;
+
+use async_trait::*;
+
+use crate::prelude::{format_err, Error, SubgraphManifest};
+use futures03::future::AbortHandle;
+
+pub mod blockchains;
+use blockchains::Blockchain;
+
+#[derive(Clone, Debug, PartialEq, Eq, Hash)]
+pub struct NetworkInstanceId {
```

Comment:
Could you add a comment explaining what such an id signifies? Is this something like `ethereum/mainnet` where `ethereum` is the `network` and `name` is the specific instance?